analysis/GreenSKU-Framework/figures/paper_figures_original/cluster_savings.txt

```diff
@@ -0,0 +1 @@
+Average cluster-level savings: 15%
```

Comment:
Instead of saving these things, I think you can just have them output in the notebook.